modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,377 @@
+
+=== Schema types translation
+
+[tabs]
+======
+Avro::
++
+--
+* Redpanda supports direct translations of the following Avro types to Iceberg value domains:
+** boolean
+** int
+** long
+** float
+** double
+** bytes 
+** string
+** record
+** array
+** maps
+** fixed
+** decimal
+** uuid
+** date
+** time
+** timestamp
+* Different flavors of timestamp and time types are all represented by the same Iceberg type, conversion will be done.
+* Avro unions are flattened to structs with optional fields, out of which only one is present, or none in case there is a field with null type:
+** For example, the union `["int", "long", "float"]` is represented as an Iceberg struct `OBJECT(0 INT NULLABLE, 1 LONG  NULLABLE, 2 FLOAT NULLABLE)`.
+** The union `["int", null, "float"]` is represented as an Iceberg struct `OBJECT(0 INT NULLABLE, 1 FLOAT NULLABLE)`.
+* All fields are required by default (Avro always sets a default in binary representation).
+* Duration logical type is ignored.
+* Null type is ignored and not represented in Iceberg schema.
+* Recursive types are not supported.
+--
+
+Protobuf::
++
+--
+* Redpanda supports direct translations of the following Avro types to Iceberg value domains: 
+** BOOL
+** DOUBLE
+** FLOAT
+** INT32
+** SINT32
+** INT64
+** SINT64
+** SFIXED32
+** SFIXED64
+** STRING
+** BYTES
+** MAP
+* Repeated values are translated into Iceberg ARRAY types.
+* Enums are translated into Iceberg INT types based on the integer value of the enumerated type.
+* UINT32 and FIXED32 are translated into Iceberg LONG types as that is the existing semantic for unsigned 32bit values in Iceberg.
+* UINT64 and FIXED64 values are translated into their base 10 string representation.
+* Timestamp well known protos are translated into TIMESTAMP Iceberg types.
+* Messages are converted into STRUCT types.
+* Recursive types are not supported.
+--
+======
+
+== Set up catalog integration
+
+You can configure the Iceberg integration to either create a file in the same object storage bucket or container to serve as the catalog, or connect to a REST-based catalog. 
+
+Set the cluster configuration property `iceberg_catalog_type` with one of the following values:
+
+* `object_storage`: Write catalog files to the same object storage bucket as the data files. Use the object storage URL to access the catalog for your Redpanda Iceberg tables.
+* `rest`: Connect to and update an Iceberg catalog hosted on a REST server.
+
+For an Iceberg REST catalog, set the following additional cluster configuration properties:
+
+* `iceberg_rest_catalog_endpoint`: The endpoint URL for your Iceberg catalog, which you are either managing directly, or is managed by an external catalog service.
+* `iceberg_rest_catalog_prefix`: Append a prefix to the catalog path, e.g. `/v1/{prefix}/namespaces`.
```

Comment:
@mmaslankaprv is this property required to use the REST catalog? And `{prefix}` in the example is a placeholder for the value set for the property? Is `namespaces` a default or standard in catalog paths?

Reply:
@mmaslankaprv in that case I think it might be better to not mention the property here as long as it's in the reference (we'll be pointing out that the full list of available catalog properties can be viewed there)